### ISSUE TITLE
Resize when hiding items?

### ISSUE THREAD

**Issue by GollyJer** [computerjazz/react-native-draggable-flatlist#198]
Hi! This is a question and not a feature request. 
Also [posted on StackOverflow](https://stackoverflow.com/questions/62616859/resize-draggableflatlist-when-rows-are-hidden). Feel free to close this if you feel that's a better place to answer.

I'm using the `isActive` property to set css `display: none` on some rows.  
The goal was to make drag/drop by top level category easier.  

However, the rows are still taking up space.  I'm hoping to (in this example) get Sleep and Energy to be right on top of each other.

<img src=https://user-images.githubusercontent.com/689204/85934452-660e3100-b897-11ea-9e4c-0041cdfb75d0.gif width=360>

Is there something simple I'm missing to get the list to resize when the rows are hidden?

Thanks!

**Comment by GollyJer** [comment]
P.S. - I also made a [react-native-draggable-flatlist](https://stackoverflow.com/questions/tagged/react-native-draggable-flatlist) `tag` on stackoverflow and tagged all the questions I could find properly.

**Comment by jacquesdev** [comment]
@GollyJer - I am struggling with something similar. I am removing an item from data, but then the correct item is not being re-rendered, in other words, and based your example, it looks like we need a way to re-render if data changes, but from what I can see this is not happening at the moment

**Comment by jacquesdev** [comment]
Check this out https://github.com/thomasrovayaz/react-native-draggable-dynamic-flatlist

**Comment by computerjazz** [comment]
Interesting. What would happen if you grabbed the lower item? both lists would collapse, but the item would still be under your finger, so there'd still be a big gap, no?

**Comment by GollyJer** [comment]
Yeah. We started only collapsing lists below the selected item for that exact reason.
In the end we made it a two step process. Tap the header to collapse all lists, then you're able to drag drop.

To get the list collapsing our workaround was to add [react-native-collapsible](https://github.com/oblador/react-native-collapsible) around the lists.
